04_numeronym/madhavansu/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprint(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+func numeronyms(vals ...string) []string {
+	strcopy := make([]string, len(vals))
```

Comment:
It's not really a copy. Use `s` for slice or `result` instead.

Reply:
Updated.